examples/react-renderer/src/Autocomplete.tsx

```diff
@@ -63,8 +64,6 @@ export function Autocomplete(
                       query,
                       params: {
                         hitsPerPage: 5,
-                        highlightPreTag: '<mark>',
-                        highlightPostTag: '</mark>',
```

Comment:
Why remove these?

Reply:
Because we have already defined "mark" as default value for tagName prop in Highlight component